lib/src/constants/command_constants.dart

```diff
@@ -1,4 +1,5 @@
 /// Stores all the commands used throughout the app that
+library;
```

Comment:
Why has this been added?

Reply:
Auto added by linter.

linter: "Dangling library doc comment. Add a 'library' directive after the library comment."

has no effect other than for the auto generated API reference (not sure what the difference is though). Would change the above comment to "//" or keep library.

Reply:
guessing it means the file will be added under "Libraries" in the attached picture with all constants documented.
![Skärmbild 2024-12-13 005419](https://github.com/user-attachments/assets/9863eaf3-4b2a-45e4-8992-6aa257e55325)